torch/_refs/__init__.py

```diff
@@ -3182,6 +3183,81 @@ def logspace(
     return prims.to_dtype(torch.pow(base, ret), dtype)
 
 
+@overload
+def meshgrid(tensors: Sequence[TensorLikeType], indexing: str):
+    pass
+
+
+@overload
+def meshgrid(*tensors: TensorLikeType, indexing: str):
+    pass
+
+
+@register_decomposition(torch.ops.aten.meshgrid)
```

Comment:
We don't need this decomposition because it's CompositeImplicitAutograd.

Reply:
So `__torch_dispatch__`/AOTAutograd/backends will never see it.

Reply:
What about in inference mode?

Reply:
I think inference mode is better served by allowing us to utilize CompositeImplicitAutograd decompositions in inference mode.

Reply:
> What about in inference mode?

Actually you're right, we won't even see these in inference mode after Python key was added to the CompositeImplicitAutograd set. Just going to keep it for now though since in the short term though its just a matter of where you skip.